plugins/cue_points/base/batch/copyCuePoints/Engine/KLiveClippingCopyCuePointEngine.php

```diff
@@ -10,6 +10,10 @@ protected static function postProcessCuePoints($copiedCuePointIds) {}
 
     protected function shouldCopyCuePoint($cuePoint)
     {
+        if (KCopyCuePointEngine::shouldCopyCuePoint($cuePoint))

```

Comment:
Parent::...

Reply:
This is intentional as the parent (liveToVod engine) allow all cue point regardless of time (to match current behavior of the batch job and I would to exec the logic of the sub-type filtering